tests/runtime/builtin

```diff
@@ -276,3 +276,10 @@ EXPECT @[{.n=0x[0-9a-f]+,.c=120}]: 1
 REQUIRES_FEATURE dwarf
 TIMEOUT 5
 BEFORE ./testprogs/uprobe_test
+
+NAME jiffies
+PROG i:ms:1 { printf("SUCCESS %d\n", jiffies); exit(); }
+EXPECT SUCCESS -?[0-9][0-9]*
```

Comment:
Is the `-?` bit of the regex cuz you're trying to print a u64 as an signed 32 bit int? Shouldn't the format specifier be `%llu` and the you can drop the `-?`?

Reply:
I think yes, other builtins like `nsecs` also has the same problem, shall fix them all later and testing 😶‍🌫️.